features/user/managing_customers/seeing_orders_statistics_on_customer_details_page.feature

```diff
@@ -0,0 +1,41 @@
+@customer_statistics
+Feature: Seeing customer's orders' statistics
+    In order to know how many orders a customer has placed, and what's their total value
+    As an Administrator
+    I want to be able to see orders statistics of customer on their details page
+
+    Background:
+        Given the store operates on a channel named "Web-US" in "USD" currency
+        And the store also operates on another channel named "Web-UK" in "GBP" currency
+        And the store has a product "Onion" priced at "$200" in "Web-US" channel
+        And this product is also priced at "£100" in "Web-UK" channel
+        And the store has customer "w.gandalf@sarumans.ok"
+        And the store also has customer "b.baggins@shire.me"
+        And this customer has placed 5 orders on the "Web-US" channel in each buying 12 "Onion" products
```

Comment:
I would move this steps to scenarios. Otherwise we have to persist all 17 order for each, even if its not in a scenario scope.